README.md

```diff
@@ -49,15 +53,57 @@ chmod +x ./deployment/build-s3-dist.sh
     s3://my-bucket-name-$REGION/$SOLUTION_NAME/$VERSION/
     ``` 
 
-## Deploying the customized solution
+### Deploying the customized solution
 * Get the link of the custom-control-tower-initiation.template loaded to your Amazon S3 bucket. 
 * Deploy the Customizations for AWS Control Tower solution to your account by launching a new AWS CloudFormation stack using the link of the custom-control-tower-initiation.template.
 
+## Deploying with GitHub as the Source
+
+### Prepare a GitHub Repository
+Create a repository within your GitHub account, and populate with the contents of this repository. 
+**Consider making the target repository private**. 
+You can use the [Import your project to GitHub](https://github.com/new/import) to make this process easier.
 
+### Running unit tests for customization 
+* Clone your GitHub repository, You might use a tool such as [GitHub Desktop](https://desktop.github.com/download/), if you wish.
+* then make the desired code changes
+* Next, run unit tests to make sure added customization passes the tests
+
+```  
+chmod +x ./deployment/run-unit-tests.sh
+./deployment/run-unit-tests.sh
+```
+Once you have a build ready, commit it to the repository and push it to GitHub.
+
+### Deploying the customized solution
+* [Developer Tools - Connections](https://console.aws.amazon.com/codesuite/settings/connections) instance for GitHub
+* Select **Create connection**
+  * Select `GitHub` as the **provider**
+  * **Create GitHub App connection** in **Connection name** type `GitHub - CfCT`
+  * Select **Connect to GitHub**
+* Connect to GitHub
+  * Select **Install a new app**
+  * Select the GitHub User/Organization for your repository
+* AWS Connector for GitHub
+  * Under **Repository access**, select **Only select repositories** and select only the repository you created earlier.
+  * Select **Save**
+* Make a note of the Code Connections ARN, as you'll need to provide this when deploying the AWS CloudFormation stack.
+
+* Get the link of the custom-control-tower-initiation.template in the root of your repository.
+* Deploy the Customizations for AWS Control Tower solution to your account by launching a new AWS CloudFormation stack using the link of the custom-control-tower-initiation.template.
```

Comment:
Consider removing `using the link of the` . Unclear reference on which link IMO.